src/material/checkbox/checkbox.ts

```diff
@@ -215,9 +219,21 @@ export class MatCheckbox implements AfterViewInit, ControlValueAccessor, Focusab
   }
 
   ngAfterViewInit() {
+    this._focusMonitor.monitor(this._elementRef, true).subscribe(focusOrigin => {
```

Comment:
I don't think we should be using the `focusMonitor` here since the only element that has focus is the `input`. We can just bind a `blur` event to it directly.

Reply:
Hey @crisbeto, We already have a blur event attached to the input, but its not appending the cdk-focused class when focused.

Reply:
That's not what the tooltip uses to show itself though. It just listens to focus events on the host element.